File: 2529. Maximum Count of Positive Integer and Negative Integer.py
class Solution(object):
    def maximumCount(self, nums):
        pos = 0
        neg = 0
        for i in range(len(nums)):
            if nums[i] > 0:
                pos += 1
            if nums[i] < 0:
                neg += 1
        return max(pos, neg)
        

File: 2529. Maximum Count of Positive Integer and Negative Integer.ts
function maximumCount(nums: number[]): number {
let pos:number=0, neg:number=0;
for (let  i:number=0;i<nums.length;i++){ if (nums[i]>0)  pos++; if (nums[i]<0) neg++; }   
return Math.max(pos,neg);    
};
